README.md

```diff
@@ -102,6 +105,9 @@ err = iter.Do(
 		if err := row.ToStruct(&rec); err != nil {
 			return err
 		}
+		if row.Replace {
+			recs = recs[:0]

```

Comment:
Should there also be a comment here?

Reply:
Yes, indeed, I've added a comment to this line